pkg/commands/commands.go

```diff
@@ -66,6 +66,10 @@ if [ ! -d ${RESULTS_DIR_PATH} ]; then
   mkdir -p ${RESULTS_DIR_PATH}
 fi
 gatling.sh -sf ${SIMULATIONS_DIR_PATH} -s %s -rsf ${RESOURCES_DIR_PATH} -rf ${RESULTS_DIR_PATH} %s
+
+if [ $? -eq 0 ]; then
```

Comment:
@yokawasa 
I feel that if the gatling-runner container fails, the COMPLETED file is not created and the gatling-result-transfer container does not terminate.
How about creating a FAILED file if gatling-runner fails and a SUCCEEDED file if it succeeds?
(I'm sure there are other ways to do this)

Reply:
@ksudate 
That's a great point ❤️ !! 
Yes, like you said, it's better to terminate the gatling-result-transfer container as quickly as possible when the runner fails. I'll take your idea to create `FAILED` file.
Thanks for the comment!

Reply:
@ksudate 

I fixed the code https://github.com/st-tech/gatling-operator/pull/65/commits/9b0b71588df5fcc26f58a2500c940681023a8464

Also I added the Test3 on the PR description. 
> Test3 - to see if gatling-result-transferer container ends as quickly as gatling-runner container fails

Please check again

Reply:
Thanks for the fix.
Looks like it's all good!